umich_api/api_utils.py

```diff
@@ -114,21 +122,44 @@ def _api_call(self, api_call, access_token, method="GET", payload=None):
             return resp
 
     def get_next_page(self, response):
+        #type: requests.Response -> Dict
```

Comment:
```suggestion
        #type: (requests.Response) -> Dict
```
i think this is causing an error if the parenthesis is not there

Reply:
Thanks! I'll have to check my debugger to see if it picks this up. I'd rather not use this style at all but Student Explorer is still Python 2.7 :(